content/docs/talks.md

```diff
@@ -0,0 +1,23 @@
++++
+title="Talks"
+weight=2
+getting-started=true
++++
+
+## Talks
+
+We love talks to share the latest development updates, covering Buildpacks basics and more, receive feedback and questions, and get to know other members of the community.
+
+### Introductory Talks  
+
+If you are new to the project, you can watch some of introductory talks on [Cloud Native Buildpacks' YouTube channel](https://www.youtube.com/@buildpacks)
```

Comment:
```suggestion
```

We have not been very good stewards of our YouTube channel. I would drop the meeting recordings below the conference talks to increase the chances that the first click will be useful.